eng/Packages.Data.props

```diff
@@ -148,6 +148,8 @@
   <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Azure.Monitor.OpenTelemetry'))">
     <!-- OpenTelemetry dependency approved for Azure.Monitor.OpenTelemetry.Exporter package only -->
     <PackageReference Update="OpenTelemetry" Version="1.6.0" />
+    <PackageReference Update="OpenTelemetry.Instrumentation.AspNetCore" Version="1.6.0" />
```

Comment:
Can we not do 1.7.0?

Reply:
I thought you had asked us to do a release first with 1.6.0 before upgrading to 1.7.0.
If I'm misremembering, we can upgrade to 1.7.0 this release.